hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -1119,7 +1128,8 @@ private void initialCommit(String createInstantTime, List<MetadataPartitionType>
 
     LOG.info("Committing " + partitions.size() + " partitions and " + totalDataFilesCount + " files to metadata");
 
-    commit(createInstantTime, partitionToRecordsMap, false);
+    commit(createInstantTime.equals(SOLO_COMMIT_TIMESTAMP) ? createInstantTime : createInstantTime + METADATA_TABLE_INIT_TIME_SUFFIX,
```

Comment:
can we write a functional test. 
start hudi table w/ default. after 2 commits, enable col stats. after 1 commit succeeds, confirm metadata table timeline is as expected. we have a delta commit for every commit in data table. and we have additional dc w/ "000" suffix. we can even check the commit metadata for this dc to ensure it has data only for col_stats partition and not for "files" partition.